src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -192,14 +192,16 @@ class Preview extends React.Component<Props> {
 
     setContent = (previewContent: string) => {
         const previewDocument = this.getPreviewDocument();
-
         if (!previewDocument) {
             return;
         }
 
+        const preservedScrollPos=this.getPreviewScrollPos();
         previewDocument.open(); // This will lose in Firefox the and safari previewDocument.location
         previewDocument.write(previewContent);
         previewDocument.close();
+        setTimeout(() => this.setPreviewScrollPos(preservedScrollPos), 100);
```

Comment:
A timeout mostly means nothing good. What happens if we directly scroll to the position?

Reply:
unfortunately nothing happend w/o the timeout.
the iframe's content needs to render the updated html first, so we need to wait a little. 
instead of waiting 100ms we should maybe get triggered by some ready-event of the iframe, but that was beyond my time.

Reply:
Is a timeout 0 (which means just push the method behind the JS event loop stack) maybe enough?

Reply:
i think i tried with 0 first.

will try again,  building my variant of 2.5 now, and will bring this PR forward hopefully. keep u updated

Reply:
I tried it with 0 and had no problems here so will go with that one.